### ISSUE TITLE
Missing store argument for blog app in default site config

### ISSUE THREAD

**Issue by jjatria** [preaction/Statocles#580]
The default config file generated by `create` seems to be missing a `store` argument for the Blog app.

In that file, the blog app [is defined as ](https://github.com/preaction/Statocles/blob/master/share/create/site.yml#L116-L118)

``` yaml
blog_app:
    $class: 'Statocles::App::Blog'
    url_root: '/blog'
```

But [the documentation for the blog app](https://metacpan.org/pod/Statocles::App::Blog#store) says that store is required, and indeed, with a fresh config file this dies:

```
$ perl -MBeam::Wire -E 'Beam::Wire->new( file => "site.yml" )->get("site")'
Missing required arguments: store at lib/Beam/Wire.pm line 552.
```
A change like the following makes the error go away:
``` diff
diff --git a/share/create/site.yml b/share/create/site.yml
index 36c6221..a560044 100644
--- a/share/create/site.yml
+++ b/share/create/site.yml
@@ -116,3 +116,4 @@ deploy:
 blog_app:
     $class: 'Statocles::App::Blog'
     url_root: '/blog'
+    store: '_posts'
```
I could put together a PR to fix this, but I'm not sure what you'd like as a default store for blogs (or if I'm doing something wrong and this is supposed to happen).

**Comment by preaction** [comment]
Ah, ha. This was what I was afraid of: The refactor I was in the middle of completely broke things (see f7c8d5b98da434d2752cec7230ed4f605c029005). It should be "blog" as the default. I'll fix that and make a v0 release along with #563 and #579.

**Comment by preaction** [comment]
Well, this is fixed because I reset the `v0` branch to before the aborted refactor (so, before the bug was introduced). I'll make a release tonight. Thanks for the report!
